README.md

```
@@ -1,17 +1,35 @@
 # tmai-api-spec
 
-Public API contracts for [tmai](https://github.com/trust-delta/tmai) — the UI contract between `tmai-core` and its UI clients (`tmai-ratatui`, `tmai-react`, and any third-party UI).
+Public API contracts for **tmai**, published as the stable interface between its private core and its public UI clients.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)